0000-template.md

```diff
@@ -21,18 +21,17 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Why are we doing this? What use cases does it support? What is the expected outcome?
 
-#Guide-level-explanation
+# Guide-level explanation
 [guide-level-explanation]: #guide-level-explanation
 
-Explain the proposal as if it were already included in the language, or in the kernel, and you were teaching it to another CC+ programmer or Cpcdos OS2.2 contributor. This usually means :
+Explain the proposal as if it was already included in the language, or in the kernel, and you was teaching it to another CC+ programmer or Cpcdos OS2.2 contributor. This usually means :
 
 - Introducing new named concepts.
 - Explaining the functionality largely through examples.
 - Explaining how CC+ programmers should *think* about the feature and how it should impact on the way they use it. The impact should be explained in as concrete a way as possible.
-- Explain how the feature fits into the kernel and its relationship to different parts of it.
+- Explaining how the feature fits into use and the relationship between its different parts.
```

Comment:
When I read that I can't figure out what you are talking about. Is it about the kernel or about the language ?